ghcide/src/Development/IDE/Main.hs

```diff
@@ -238,7 +238,15 @@ defaultArguments recorder plugins = Arguments
             { optCheckProject = pure $ checkProject config
             , optCheckParents = pure $ checkParents config
             }
-        , argsLspOptions = def {LSP.optCompletionTriggerCharacters = Just "."}
+        , argsLspOptions = def
+            { LSP.optCompletionTriggerCharacters = Just "."
+            -- Generally people start to notice that something is taking a while at about 1s, so
+            -- that's when we start reporting progress
+            , LSP.optProgressStartDelay = 1_00_000
```

Comment:
What a weird number. Shouldn't this be `1_000_000`

Reply:
:facepalm: